package.json

```diff
@@ -25,35 +25,28 @@
         "ocamlfind",
         "sexplib"
       ]
-    },
-    "buildsInSource": "_build"
+    }
   },
   "dependencies": {
-    "@opam/alcotest": "0.8.3",
-    "@opam/astring": "0.8.3",
-    "@opam/bisect_ppx": "1.3.4",
-    "@opam/bos": "0.2.0",
-    "@opam/cmdliner": "1.0.2",
-    "@opam/cppo": "1.6.5",
-    "@opam/dune": "1.3.0",
-    "@opam/fpath": "0.7.2",
-    "@opam/js-build-tools": "113.33.04",
-    "@opam/lambdasoup": "0.6.3",
-    "@opam/markup": "*",
-    "@opam/oasis": "0.4.11",
-    "@opam/ocamlfind": "1.8.0",
-    "@opam/result": "1.3",
-    "@opam/sexplib": "*",
-    "@opam/tyxml": "4.3.0",
-    "ocaml": "~4.2.0"
-  },
-  "resolutions": {
-    "@opam/markup": "aantron/markup.ml:markup.opam#9f8e77",
-    "@opam/sexplib": "bryphe/sexplib#548f94d"
+    "@opam/bisect_ppx": "^1.3.4",
+    "@opam/bos": "*",
+    "@opam/dune": "^1.3.0",
+    "@opam/tyxml": "^4.3.0",
+    "ocaml": "*"
```

Comment:
Okay, I think the version should be `^4.2.0` then so it's any "patch" over `4.2.0`.

Reply:
It's because of what I wrote in the other discussion, sexplib (even in the resolution) is depending on sexplib0 that in turn needs >= 4.4.0. If someone can find a set of resolutions to make this work I would love it but I can't get it to work without forking a bunch of libraries.